brokerapi/brokers/brokers_test.go

```diff
@@ -178,84 +174,54 @@ var _ = Describe("Brokers", func() {
 
 		})
 
-		It("should have 1 cloudsql plan available", func() {
-			serviceList := gcpBroker.Services()
+		It("should have 15 cloudsql plans available", func() {
+			serviceList, err := gcpBroker.Services(context.Background())
+			Expect(err).ToNot(HaveOccurred())
 			for _, s := range serviceList {
 				if s.ID == serviceNameToId[models.CloudsqlMySQLName] {
-					Expect(len(s.Plans)).To(Equal(1))
+					Expect(len(s.Plans)).To(Equal(15))
 				}
 			}
 
 		})
 
-		It("should have 1 bigtable plan available", func() {
-			serviceList := gcpBroker.Services()
+		It("should have 2 bigtable plans available", func() {
+			serviceList, err := gcpBroker.Services(context.Background())
+			Expect(err).ToNot(HaveOccurred())
 			for _, s := range serviceList {
 				if s.ID == serviceNameToId[models.BigtableName] {
-					Expect(len(s.Plans)).To(Equal(1))
+					Expect(len(s.Plans)).To(Equal(2))
 				}
 			}
 
 		})
 
 		It("should have 1 debugger plan available", func() {
-			serviceList := gcpBroker.Services()
+			serviceList, err := gcpBroker.Services(context.Background())
+			Expect(err).ToNot(HaveOccurred())
 			for _, s := range serviceList {
 				if s.ID == serviceNameToId[models.StackdriverDebuggerName] {
 					Expect(len(s.Plans)).To(Equal(1))
 				}
 			}
 		})
 
-		It("should error if plan ids are not supplied", func() {
```

Comment:
This resolution is no longer handled at this point in the code. It's caught on startup and will fail early rather than later (same with the below portion of the SQL)